### PR TITLE
Update build image in release/6.0

### PR DESCRIPTION
**Pull request by dreddy-work**
Related to #3328

### CODE REVIEW

**Comment by mmitche** [comment]
Do not merge til Jan (this has laready been fixed internally)

**Review by mmitche — APPROVED** [review]

**Review by ViktorHofer — APPROVED** [review]

**Comment by dreddy-work** [comment]
@mmitche , can merge this if it is okay now?

**Pull request merged**
